adapters/Laravel/Vendor/src/Repositories/EloquentVendorRepository.php

```
@@ -1,27 +1,197 @@
+        ]);
+
+        if (($approvalRecord = $vendor->getApprovalRecord()) !== null) {
+            $model->approved_by_user_id = $approvalRecord->getApprovedByUserId();
+            $model->approved_at = $approvalRecord->getApprovedAt();
+            $model->approval_note = $approvalRecord->getApprovalNote();
+        }
+
+        $model->save();
+
+        return $model;
+    }
```

PHPMD (2.15.0)

[error] Avoid assigning values to variables in if clauses and the like (line '91', column '14'). (undefined)

(IfStatementAssignment)